internal/observability/attributes.go

```diff
@@ -2,6 +2,16 @@ package observability
 
 const (
 	ModuleNameAttribute          = "ftl.module.name"
-	StatusSucceededAttribute     = "ftl.status.succeeded"
+	OutcomeStatusNameAttribute   = "ftl.outcome.status"
 	RunnerDeploymentKeyAttribute = "ftl.deployment.key"
+
+	SuccessStatus = "success"
+	FailureStatus = "failure"
 )
+
```

Comment:
What do you think about reprising the pattern you came up with ages ago and adding a helper function here that just returns `attribute.String(observability.OutcomeStatusNameAttribute, observability.SuccessOrFailureStatus(succeeded))`? Since so much of the code that uses this now is basically just duplicating that line

Reply:
sgtm, applied

Reply:
micro-nit: could we rename `SuccessOrFailureStatus` to `SuccessOrFailureStatusAttr` since that's what it returns?